tests/rkt_fetch_test.go

```diff
@@ -86,7 +87,7 @@ func testFetchFromFile(t *testing.T, arg string, image string) {
 // TestFetch tests that 'rkt fetch/run/prepare' for any type (image name string
 // or URL) except file:// URL will work with the default, store only
 // (--store-only) and remote only (--no-store) behaviors.
-func TestFetch(t *testing.T) {
+func TestFetchOnly(t *testing.T) {
```

Comment:
There are other `TestFetch*` methods here, so calling `TestFetch` in isolation was not possible, i.e. `make functional-check GO_TEST_FUNC_ARGS='-run TestFetch'` also calls `TestFetchFromFile`, `TestFetchFullHash`, ...

I should have made this more explicit in the actual commit message. I can revert this to the old name, but it is just a little easier when invoking this test case in isolation locally.